lib/cli.js

```diff
@@ -123,6 +123,14 @@ function init() {
       return;
     }
 
+    if (cmd.startsWith('generator-')) {
```

Comment:
Maybe it'd be better to just run the generator in this case? If we know the user meant `yo foo` instead of `yo generator-foo`, I feel like we should go ahead and follow the user intent.